### ISSUE TITLE
VRRAP: Update Form Page Content

### ISSUE THREAD

**Issue by dneel-bah** [department-of-veterans-affairs/va.gov-team#22658]
As a developer, I need to update the VRRAP form page content so that it is inline with DEPO plain language patterns and practices.

## Assumptions:
1. If the content in the prototype does not match the content doc, the content doc is to be followed.
2. The chapter 2 title of "Review Application" is out of scope due to it being a shared component.
3. The pages match the layout and styling of the prototype as closely as possible.

## Acceptance Criteria:
1. The form title is: "Apply for the Veteran Rapid Retraining Assistance Program (VRRAP)"
2. The chapter 1 title is: "VRRAP application"
3. The "Confirm you're eligible for VRRAP" section is:
    a. not displayed in chapter 1.
    b. displayed on review page.
4. Review page of the form matches the content and layout as specified in SA1.

## Supporting Artifacts:
1. [Content Doc](https://github.com/department-of-veterans-affairs/va.gov-team/files/6266478/vrrap_form_content_updates.txt)
2. Prototype: 
   a. Unauthenticated user: https://bahdigital.invisionapp.com/share/9UIAGJUQNXC
   b. Authenticated user: https://bahdigital.invisionapp.com/share/RMIAGJTSDEA

**Comment by zurbergram** [comment]
AC 3,4,5,6 are being resolved in second PR for https://github.com/department-of-veterans-affairs/va.gov-team/issues/22315

**Comment by zurbergram** [comment]
AC 1,2 PR https://github.com/department-of-veterans-affairs/vets-website/pull/16669

**Comment by zurbergram** [comment]
AC 3,4 https://github.com/department-of-veterans-affairs/vets-website/pull/16670

**Comment by delli-gatti-michael** [comment]
local testing complete

**Comment by delli-gatti-michael** [comment]
[Testrail test case](https://dsvavsp.testrail.io/index.php?/runs/view/1652&group_by=cases:section_id&group_order=asc)

**Comment by delli-gatti-michael** [comment]
Currently blocked by #22925

The above AC will need to be tested on Staging before this story is closed

**Comment by delli-gatti-michael** [comment]
Changes are on Staging. Closing this issue
